src/app/main.cpp

```diff
@@ -86,6 +86,11 @@ int main( int argc, char **argv )
     return 0;
   }
 
+#if QT_VERSION >= QT_VERSION_CHECK( 6, 5, 1 )
+  // Avoids systematic crash after first launch, a regression in Qt >= 6.5.1
```

Comment:
Link to issue or similar?

Reply:
@m-kuhn , it's not reported on Qt yet (it's a new crash on 6.5.1, and it's not in there: https://bugreports.qt.io/browse/QTBUG-114039?jql=affectedVersion%20%3D%206.5.1%20AND%20text%20~%20%22qml%20crash%22%20ORDER%20BY%20created%20DESC ). I'll make it my mission to try 6.5.2+ ;)